createNPMPackage.sh

```diff
@@ -7,8 +7,24 @@ ROOT=$(pwd)
 # unset CI so all archs are built
 unset CI
 
+rm -rf ./android/src/main/jniLibs
```

Comment:
I think it's time to make the script more generic. Can you make an array with RN versions and just iterate over them?

Reply:
It's a good idea 👍 I'll do it.